### PR TITLE
Add doctest and link checks

### PR DESCRIPTION
**Pull request by JoelPasvolsky**

### CODE REVIEW

File: docs/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-breathe<4.13.0
+breathe>4.14.0
```
Comment (randomir) [inline docs/requirements.txt]: we should probably bound sphinx here as well

Comment (JoelPasvolsky) [reply docs/requirements.txt]: Done, thanks!

**Review by randomir — APPROVED** [review]

**Comment by JoelPasvolsky** [comment]
Why is Py3.5 still being tested here?

**Comment by randomir** [comment]
PR to drop py35 coming in 10.

**Comment by randomir** [comment]
Once #183 is merged, I'll rebase this PR (unless you'd prefer to do that, @JoelPasvolsky?)

**Comment by JoelPasvolsky** [comment]
Either option is okay by me, @randomir.

**Comment by JoelPasvolsky** [comment]
@randomir, rebased

**Pull request merged**